### DISCUSSION TITLE
Auto Bed Mesh with CR Touch and Sprite

### DISCUSSION THREAD

**Discussion by kjm8844** [mriscoc/Ender3V2S1#962] [Q&A]
I have been trying to run the auto bed mesh on my ender 3 max neo for a while and have been having issues where it will do the first handful of points fine and then it will not deploy the probe and the nozzle will hit the bed. I have done the z-probe offset and bed tramming and both are good. If anyone has any ideas please let me know.

**Comment by mriscoc** [comment]
Turn off HS mode.

**Comment by kjm8844** [comment]
I did do this and I got the cr touch and sprite to stop hitting the bed. However I have a new problem now. If I run the auto bed mesh it will take 5-7 points and then on the 8th or so point the z will not raise and the probe will not raise for the next point and all points after will be extremely high.
